app/controllers/application_controller.rb

```diff
@@ -1,2 +1,61 @@
+# frozen_string_literal: true
+
 class ApplicationController < ActionController::Base
+  include CurrentUser
+  include JsonResolver
+
+  protect_from_forgery unless: -> { request.format.json? }
+
+  rescue_from Exception do |e|
+    error(e)
+  end
+
+  # 400
+  rescue_from Exceptions::BadRequest, ActiveRecord::RecordInvalid, ActionController::ParameterMissing, ArgumentError do |e|
+    exception_handler e, :bad_request
```

Comment:
에러 클래스별로 http_status 메소드를 구현해두셨던데 그걸 사용하면 좋을 것 같네요
```suggestion
    exception_handler e, e.class.http_status
```

Reply:
[ef05ddc](https://github.com/travelingaries/taxi-dispatch-backend-api/pull/1/commits/ef05ddc30734f86914835d18edc1bc045454d8a3)에서 수정했습니다